src/hooks/index.ts

```diff
@@ -35,3 +36,31 @@ export const useCard = () => {
 
   return [card, setNewCard] as const;
 };
+
+export const useCardInputRefs = () => {
+  const cardNumberRef = useRef<HTMLInputElement>(null);
+  const expiredDateRef = useRef<HTMLInputElement>(null);
+  const ownerNameRef = useRef<HTMLInputElement>(null);
+  const cvcRef = useRef<HTMLInputElement>(null);
+  const password1Ref = useRef<HTMLInputElement>(null);
+  const password2Ref = useRef<HTMLInputElement>(null);
+
+  const inputRefs = [
+    cardNumberRef,
+    expiredDateRef,
+    ownerNameRef,
+    cvcRef,
+    password1Ref,
+    password2Ref,
+  ];
```

Comment:
여기서 refs 객체를 핸들링하는 부분이 조금 아쉬운데요, 팁을 하나 드릴게요! 이건 적용해보는 게 라잇한테 도움이 많이 될 거에요.

```suggestion
export const useCardInputRefs = () => {
  const inputRefs = Array.from({ length: 6 }).map(createRef<HTMLInputElement>);
```